docs/components/tasklist/userguide/using-tasklist.md

```diff
@@ -56,6 +56,20 @@ Select the **Assigned to me** list to see the tasks that are assigned to you. Se
 
 ![tasklist-claimed-by-me-list](img/tasklist-claimed-by-me-list_light.png)
 
+Tasklist users can now receive a browser notification when new tasks are assigned to them:
+
+![Tasklist notifications banner](./img/tasklist-notifications.png)
+
+The **Don't miss new assignments** banner at the top of the page appears when the user either assigns a new task to themselves, or opens a task that is already assigned to them. To turn on browser notifications, click **Turn on notifications**.
+
+You can turn these notifications off and on by clicking the profile icon in the top right corner of the screen. Toggle the **Browser notifications** setting to receive notifications about new task assignments or turn off notifications:
```

Comment:
That's not implemented in 8.5. For now you will have to go into your browser settings and disable notifications for that site.

For example, in Google Chrome, you click the icon in the left of the address bar and turning off notifications.

Reply:
Adjusted 👍